server/odc-core/src/main/java/com/oceanbase/odc/core/shared/constant/AuditEventType.java

```diff
@@ -74,6 +74,8 @@ public enum AuditEventType implements Translatable {
 
     PROJECT_MANAGEMENT,
 
+    SQL_SECURITY_RULES_MANAGEMENT
```

Comment:
We are more inclined to use a single term such as `SQL_SECURITY_RULE_MANAGEMENT`